SA/sa.typ

```diff
@@ -3,13 +3,25 @@
 #set text(..sa_text_style)
 #set heading(..sa_header_style)
 
-#sa_title_page(
-  "VisualFP",
-  "Designing a Visual, Block-Based Environment to Create & Execute Haskell Code");
+#let metadata = (
+  title: [VisualFP],
+  description: [Designing a Visual, Block-Based Environment to Create & Execute Haskell Code],
+  organization: [Department of Computer Science \ OST - University of Applied Sciences \ Campus Rapperswil-Jona],
+  thesis: [Student Thesis],
+  term: [Autumn Term 2023],
+  authors: [Lukas Streckeisen, Jann M. Flepp],
+  authors-short: [L. Streckeisen, J. Flepp],
+  advisor: [Farhad D. Mehta],

```

Comment:
```suggestion
  advisor: [Prof. Dr. Farhad D. Mehta],

```
I think in an academic environment this is kind of important ;)

Reply:
Yes, it is 😊

Adjusted it accordingly